VERSION

```diff
@@ -1 +1 @@
-v7
+v8
```

Comment:
Please do not set the version yourself, please revert this change from your commits.

Reply:
Done, thanks